computable/tests/conftest.py

```diff
@@ -30,13 +31,13 @@ def ether_token(w3, ether_token_opts):
         abi = json.loads(f.read())
     with open(os.path.join(contract_path, 'ethertoken.bin')) as f:
         bc = f.read()
-    token = w3.eth.contract(abi=abi, bytecode=bc.rstrip('\n'))
-    tx_hash = token.constructor(w3.eth.defaultAccount,
+    t = w3.eth.contract(abi=abi, bytecode=bc.rstrip('\n'))
```

Comment:
Style nit, I think `token` is more pythonic than `t` here, but not a strong opinion.